### DISCUSSION TITLE
What is target release date for OpenSSL 3.3?

### DISCUSSION THREAD

**Discussion by xnox** [openssl/openssl#23066] [Q&A] [answered]
What is target release date for OpenSSL 3.3?

**Comment by t8m** [comment, marked as answer]
We've announced quite a while ago that we are switching to time-based release policy here: https://www.openssl.org/blog/blog/2023/08/27/steps-forward/

This blog post also announced that the first release that will be under the time-based release policy will be the 3.3 version to be released in April.
